portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,172 @@
+          Although I had honed many skills and built rapport with customers, I wanted education.  <br>
+          So, I decided I was ready to return to an academic environment.<br><br>
+          I left my position at Fantasy in [2018] and enrolled at the Borough of Manhattan Community College. <br>
+          Over the last two years I have continued to build on my knowledge, gained new friends, and have matured in ways that leave me inspired for the future.<br><br>
+          I seek to further my career in computer science and mathematics, and the opportunity to learn in a challenging environment, and to work with others at the top of my field.
+          <br>
+          <h2>Some of my Projects</h2>
+          <ul>
+              <li>
+                  <a href="https://docs.google.com/document/d/155bCb1lZvJi3AhRsqjVXlA0dqvYj0v5I_xr9Tr4YZSc/edit?usp=sharing"
+                  ><h3>Cryptography workshop I wrote the Java portion of, and taught</h3></a>
+              </li>
+              <li>
+                  <h3>Alexa based interface for school directory</h3>
+                  <div id="AlexaProjects" style="margin-left: 40px">
+                  <a href="https://github.com/andrewrubinstein/AlexaWs">
+                    Web Service that interfaces with Alexa
+                  </a><br>
+                  <a href="https://github.com/andrewrubinstein/BMCCDirectoryWebService">
+                    Web Service that interfaces with database
+                  </a><br>
+                  <a href="https://github.com/andrewrubinstein/Directory">
+                    Java Application to manage data stored in database
+                  </a>
+                  </div>
+              </li>
+              <li>
+                  <a href="https://github.com/andrewrubinstein/AscalLang"
+                  ><h3>A Programming Language I'm developing in C++ called ASCAL lang</h3></a>
+              </li>
+              <li>
+                  <a href="https://github.com/andrewrubinstein/Tetris-v1"
+                  ><h3>A Java based Tetris implementation</h3></a>
+              </li>
+              <li>
+                  <a href="https://github.com/andrewrubinstein/SnakeGame"
+                  ><h3>A Java based Snake Game with multiple Pathfinding algorithms implemented for AI</h3></a>
+              </li>
+              <li>
+                  <a href="https://github.com/andrewrubinstein/PHPClassSchedulingApp"
+                  ><h3>A Scheduling algorithm written in PHP</h3></a>
+              </li>
+          </ul>
+      </p>
+
+         
+<h2>What I enjoy photographing</h2>
+<p style="margin-left: 40px">
+    As you may notice from the collection below I most enjoy landscape photography.<br>
+    I also enjoy taking selfies, especially with larger groups of friends, being tall really helps there.<br>
+    I hope you enjoy the collection I've put together!
+</p>
+      <h2>Photos from my adventures</h2>
```

Comment:
Nit: Try to indent consistently throughout the file. Most of it is really good but this chunk looks out of place.